app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -2710,20 +2727,29 @@ protected function getProductUrlSuffix($storeId = null)
 
     /**
      * @param array $rowData
-     * @return string
+     * @return string|bool
      * @since 100.0.3
      */
     protected function getUrlKey($rowData)
     {
         if (!empty($rowData[self::URL_KEY])) {
             return strtolower($rowData[self::URL_KEY]);
         }
+        
+        /**
+         * If the product already exists, do not overwrite its
+         * URL Key with a value generated from the provided name.
+         * @see https://github.com/magento/magento2/issues/17023
```

Comment:
Comments are good, can you please remove Magento issues links from the code. Let keep it clean.